Glossary_of_Terms.txt

```diff
@@ -89,6 +89,18 @@ Player Character at a time.
 
 RPG: 
     Role-Playing Game, such as Dungeons and Dragons or Compound X!
+    
+Roll - Standard:
+    Roll 1d100.
+    
+Roll - Advantage:
+    Roll twice. Take the highest of the two rolls. 
```

Comment:
Should we add a "See the Primary Game Mechanic section of the Basic Rules" note in the description of each of these rolls? Not a blocking issue.

Reply:
That could be nice. I can do that.